www/src/components/sidebar/sticky-responsive-sidebar.js

```diff
@@ -134,9 +131,11 @@ class StickyResponsiveSidebar extends Component {
             <ChevronSvg
               size={16}
               cssProps={{
-                transform: `translate(${iconOffset}px, 5px) rotate(90deg)`,
+                transform: `translate(px, 5px) rotate(90deg)`,
```

Comment:
Looks (!) like we lost the `${iconOffset}` here?

Reply:
Woah, weird, thanks for catching that!